tutorial/lesson-1/README.md

```diff
@@ -240,3 +241,9 @@ pull in the wab UI JavaScript.
 ```
 
 Now when wabur is run the frame identifies the page as your blog.
+
+## Shortcut
+
+A script is available in the wabur gem that will execute all the steps in this
```

Comment:
scripts won't be bundled into the gem.. they're hosted at the repo only.. (as of this writing..)

Reply:
Easily changed although I'm not sure if that is best. Any preference?

Reply:
well.. even if the script was bundled in the gem, the user would have to enter the correct path to the gem to locate the script via shell command..

Reply:
The other alternative is to clone the repo, install dev-dependencies, run `build-export` and then the lesson script..